### ISSUE TITLE
.gprj file not found

### ISSUE THREAD

**Issue by Nikita7131** [kiiisy/camera#1]
in the article ( https://qiita.com/T_ksy/items/71e269da4fa06ec5c024 ) was said that the project was written in Gowin EDA, but I cannot run the project, I suspect that the .gprj file is missing, can you please add it ?

**Comment by kiiisy** [comment]
Thank you for letting me know.
Project file has been registered, please check it.

**Comment by Nikita7131** [comment]
yes, this file appeared, but the PinFloor file is still missing for launch, can you add it, please ?

**Comment by kiiisy** [comment]
The PinFloor file is the camera.cst file in the src folder.
It is already registered, so please use this one.

**Comment by Nikita7131** [comment]
![png](https://github.com/user-attachments/assets/03b807cc-14ff-42ba-90b7-473747879017)
 yes, sorry, the file is there, but for some reason ide does not see it ( (System Verilog 2017 is inserted in the settings)

**Comment by kiiisy** [comment]
I'm getting an error on my end as well.
So, the synthesis went through with the following settings.
Try it with this setup
・Set the top module
![image](https://github.com/user-attachments/assets/86fe15eb-7001-4d84-a70a-cb81f9b673ed)
![image](https://github.com/user-attachments/assets/288f0846-dc4f-446f-94ef-2d282f2513f7)
![image](https://github.com/user-attachments/assets/c9a33f85-8b06-40f1-8c08-02d7533b3003)

**Comment by Nikita7131** [comment]
Sorry for the long answer, the project has compiling and working. The only thing is that for some reason there are black stripes on the screen, I'm waiting for a new camera board to solder it with wires, instead of using connectors ...

![photo_2024-10-12_13-10-53](https://github.com/user-attachments/assets/a8510aaf-9dfa-4eb6-90d9-7b9915f25d57)
